kratos/utilities/variable_utils.h

```diff
@@ -1177,7 +1177,7 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
 
         const auto &r_communicator = rModelPart.GetCommunicator();
 
-        using ReductionType = typename std::conditional< std::is_scalar<TDataType>::value , SumReduction<double> , Array3Reduction >::type;
+        using ReductionType = typename std::conditional< std::is_scalar<TDataType>::value , SumReduction<double> , SumReduction<array_1d<double,3>> >::type;
```

Comment:
Actually, ```std::conditional``` is not needed here anymore.

Reply:
yes I wanted to comment that
it was a necessary hack at the time, can you please remove it now?